src/app/auth/user-login/_components/registration-form.tsx

```
@@ -0,0 +1,188 @@
+              <div className="space-y-2">
+                <FormField
+                  control={form.control}
+                  name="address"
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel>Address</FormLabel>
+                      <Input
+                        placeholder="Your Address"
+                        className="h-12 w-full"
+                        {...field}
+                      ></Input>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)